platform/linux_platform.go

```diff
@@ -1176,6 +1219,11 @@ func (p linux) MigratePersistentDisk(fromMountPoint, toMountPoint string) (err e
 	return
 }
 
+func (p linux) SetPersistentDiskMigrationReadahead(readahead int) error {
+	p.options.PersistentDiskMigrationReadahead = readahead
```

Comment:
i dont think this will properly work since readahead will be assigned to a copy of p.options (p isnt a pointer). have you seen it work on a live machine?